ui/src/containers/NodeVolumes.js

```diff
@@ -34,6 +34,7 @@ import {
   STATUS_FAILED,
   STATUS_AVAILABLE,
   STATUS_BOUND,
+  STATUS_READY,
```

Comment:
I think you can remove `STATUS_AVAILABLE` from the import here: it's no longer used in this file.